src/commands/hpttl.json

```
@@ -0,0 +1,85 @@
+{
+    "HPTTL": {
+        "summary": "Returns the remaining time to live (in milliseconds) of a hash key's field(s) that have an associated expiration.",
```

nit - for consistency with [hpexpiretime.json](https://github.com/valkey-io/valkey/pull/2089/files#diff-24c4eee9c3e2c2e2264fbb8716e02c48dff0b85f71c68a4e95e7e818594b1e1d)
```suggestion
        "summary": "Returns the remaining time to live in milliseconds of a hash key's field(s) that have an associated expiration.",
```